app/models/BannerTests.scala

```diff
@@ -0,0 +1,64 @@
+package models
+
+import enumeratum.{CirceEnum, Enum, EnumEntry}
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.auto._
+import io.circe.{Decoder, Encoder}
+
+import scala.collection.immutable.IndexedSeq
+
+sealed trait OphanProduct extends EnumEntry
+
+object OphanProduct extends Enum[OphanProduct] with CirceEnum[OphanProduct] {
+  override val values: IndexedSeq[OphanProduct] = findValues
+
+  case object CONTRIBUTION extends Product
+  case object RECURRING_CONTRIBUTION extends Product
+  case object MEMBERSHIP_SUPPORTER extends Product
+  case object MEMBERSHIP_PATRON extends Product
+  case object MEMBERSHIP_PARTNER extends Product
+  case object DIGITAL_SUBSCRIPTION extends Product
+  case object PRINT_SUBSCRIPTION extends Product
+}
+
+case class BannerVariant(
+  name: String,
+  titles: Option[List[String]],
+  leadSentence: Option[String],
+  messageText: String,
+  closingSentence: Option[String],
+  ctaText: String,
+  buttonCaption: String,
+  linkUrl: String,
+  hasTicker: Boolean,
+  tickerHeader: Option[String],
+  signInUrl: String,
+  secondaryLinkUrl: Option[String],
+  secondaryLinkLabel: Option[String],
+  subsLinkUrl: Option[String]
+)
+
+case class BannerTest(
+  name: String,
+  nickname: Option[String],
+  campaignCode: String,
+  pageviewId: String,
+  isOn: Boolean,
+  isHardcodedFallback: Boolean,
+  minArticlesBeforeShowingBanner: Int,
+  userCohort: UserCohort,
+  bannerModifierClass: Option[String],
+  products: Option[List[OphanProduct]],
+  locations: List[Region] = Nil,
+  variants: List[BannerVariant],
+  template: String,
```

Comment:
not sure what this is, but it can go

Reply:
This is a placeholder based on the [requirement](https://docs.google.com/document/d/1jgc8nK7fognfXan9OB_g5EOQywOPF4roNKP_3T4Na8Y):
```"Select a banner template 
Display a list of max 3 templates of existing banners 
Select one 
```

Happy to take it out but need to understand the requirement in more detail to suggest a solution

Reply:
update: discussed with @tomrf1  - we are not going to do this for MVP so will remove the field for now.